docs/Getting Started/Getting-Started-From-Scratch.md

```diff
@@ -1,108 +1,1150 @@
+    ```
+    $ mkdir new-node-1t
+    $ cd new-node-1t
+    $ java -jar ../tessera.jar -keygen -filename new-node-1
+    Enter a password if you want to lock the private key or leave blank
+    
+    Please re-enter the password (or lack of) to confirm
+    
+    10:32:51.256 [main] INFO  com.quorum.tessera.nacl.jnacl.Jnacl - Generating new keypair...
+    10:32:51.279 [main] INFO  com.quorum.tessera.nacl.jnacl.Jnacl - Generated public key PublicKey[pnesVeDgs805ZPbnulzC5wokDzpdN7CeYKVUBXup/W4=] and private key REDACTED
+    10:32:51.624 [main] INFO  c.q.t.k.generation.FileKeyGenerator - Saved public key to /Users/krish/fromscratch/new-node-1t/new-node-1.pub
+    10:32:51.624 [main] INFO  c.q.t.k.generation.FileKeyGenerator - Saved private key to /Users/krish/fromscratch/new-node-1t/new-node-1.key
+    ```
+3. Create new configuration file with newly generated keys referenced. Name it `config.json` as done in this example
+    ```
+    vim config.json
+    {
+       "useWhiteList": false,
+       "jdbc": {
+           "username": "sa",
+           "password": "",
+           "url": "jdbc:h2:/yourpath/new-node-1t/db1;MODE=Oracle;TRACE_LEVEL_SYSTEM_OUT=0",
+           "autoCreateTables": true
+       },
+       "serverConfigs":[
+           {
+               "app":"ThirdParty",
+               "enabled": true,
+               "serverAddress": "http://localhost:9081",
+               "communicationType" : "REST"
+           },
+           {
+               "app":"Q2T",
+               "enabled": true,
+                "serverAddress":"unix:/yourpath/new-node-1t/tm.ipc",
+               "communicationType" : "REST"
+           },
+           {
+               "app":"P2P",
+               "enabled": true,
+               "serverAddress":"http://localhost:9001",
+               "sslConfig": {
+                   "tls": "OFF"
+               },
+               "communicationType" : "REST"
+           }
+       ],
+       "peer": [
+           {
+               "url": "http://localhost:9001"
+           },
+           {
+               "url": "http://localhost:9003"
+           }
+       ],
+       "keys": {
+           "passwords": [],
+           "keyData": [
+               {
+                   "privateKeyPath": "/yourpath/new-node-1t/new-node-1.key",
+                   "publicKeyPath": "/yourpath/new-node-1t/new-node-1.pub"
+               }
+           ]
+       },
+       "alwaysSendTo": []
+    }
+    ```
+    
+4. If you want to start another Tessera node, please repeat step 2 & step 3
+    ```
+    $ cd ..
+    $ mkdir new-node-2t
+    $ cd new-node-2t
+    $ java -jar ../tessera.jar -keygen -filename new-node-2
+    Enter a password if you want to lock the private key or leave blank
+    
+    Please re-enter the password (or lack of) to confirm
+    
+    10:45:02.567 [main] INFO  com.quorum.tessera.nacl.jnacl.Jnac
```

Comment:
The `privateFor` array in the private-contract.js (below) must contain the tm public key of an existing tessera node.
Replace the `privateFor` key below with the key you generated for node2.
```
$ cat new-node-2t/new-node-2.pub
CziOcGtPppFyngwMz9HXyannocfCOgInG2DiJJz8A1s=
## place you key in the privateFor array.
privateFor: ["CziOcGtPppFyngwMz9HXyannocfCOgInG2DiJJz8A1s="]
```
if you do not have an existing public key in the array, you will see the following error when the script in loaded (next step).
```
loadScript("private-contract.js")
err creating contract Error: Non-200 status code: &{Status:400 Bad Request StatusCode:400 Proto:HTTP/1.1 ProtoMajor:1 ProtoMinor:1 Header:map[Date:[Mon, 17 Jun 2019 15:23:53 GMT] Content-Type:[text/plain] Content-Length:[73] Server:[Jetty(9.4.z-SNAPSHOT)]] Body:0xc01997a580 ContentLength:73 TransferEncoding:[] Close:false Uncompressed:false Trailer:map[] Request:0xc019788200 TLS:<nil>}
```